ontology/eco.md

```diff
@@ -63,5 +64,3 @@ ECO was originally created around the year 2000 to support gene product annotati
 
 ***
 For **advice on requesting new terms**, please see **[the Evidence & Conclusion Ontology wiki](https://github.com/evidenceontology/evidenceontology/wiki/New-term-request-how-to)**.
-
-This work is made possible by **[award number 1458400](http://www.nsf.gov/awardsearch/showAward?AWD_ID=1458400)** from the **US National Science Foundation's Division of Biological Infrastructure**.
```

Comment:
note that this is now redundant of the structured information, which is displayed much more nicely.